src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -33,7 +33,7 @@ internal void OnStepClosed(string name, StepTypes stepType)
         /// </summary>
         public int TotalTooltips { get; set; }
 
-        public enum StepTypes{ SURVEY, TOOLTIP, WELCOME, EXIT_TOUR };
+        public enum StepTypes{ SURVEY, TOOLTIP, LIBRARY, WELCOME, EXIT_TOUR };
```

Comment:
it doesn't matter at the moment because this is not released yet (is it?) but when you modify public enums, add the new enums to the end - because enums are compiled to ints - the order of existing integers will be modified otherwise - this will affect older clients by getting the wrong enum value.

Reply:
I stepped back and now I'm using a flag to determine if the step needs to show or hide the LibraryView.
Thanks for the comment.